### DISCUSSION TITLE
Downloading a folder?

### DISCUSSION THREAD

**Discussion by jgutierrezre** [jupyterlite/jupyterlite#891] [Q&A]
Is there a way to archive a folder to easily download it? I know I can select multiple files and download them, but this gets cumbersome quickly.

**Comment by oscarpereztuta** [comment]
Hi @jgutierrezre, this snipped will solve your need:

1. List your folders:
```python
import os
os.getcwd()
os.listdir('/drive')
```

2. Then zip `your_folder`:
```
import shutil
shutil.make_archive('/drive/your_folder', 'zip', '/drive/your_folder')
```
